packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -82,36 +81,23 @@ const ResetPassword = ({
     navigate('confirm_reset_password', { replace: true });
   }, [navigate]);
 
-  // Set tooltip error message
-  // const setErrorMessage = (errorText: string) => {
-  //   setEmailErrorText(errorText);
-  // };
-
-  // --TODO: Complete onSubmit handling in FXA-6123--
-  // Verify if provided email is linked to existing account
-  // If yes, logViewEvent (reset-password.submit),
-  // initiate password reset and send password reset link by email
-  // Navigate to confirm_reset_password
-  // Catch errors:
-  // If 'UNKNOWN_ACCOUNT' --> tooltip ('Unknown account. Sign up' w/link), log error
-  // If USER_CANCELLED_LOGIN --> log error (login.cancelled)
-  // If other error --> alertBar, ''Sorry, there was a problem resetting your password'
-  const onSubmit = () => {
+  const onSubmit = async () => {
     try {
-      // TODO resetPassword function in FXA-6123
-      account.resetPassword(email);
+      await account.resetPassword(email);
       navigateToConfirmPwReset();
-    } catch (e) {
-      const errorResetPassword = ftlMsgResolver.getMsg(
-        'reset-password-error-general',
-        'Sorry, there was a problem resetting your password'
-      );
-      alertBar.error(errorResetPassword);
+    } catch (err) {
+      if (err.errno) {
```

Comment:
This error message should probably appear in a `Banner` component vs Input tooltip.

![image](https://user-images.githubusercontent.com/22231637/217094408-782d47b4-7ef5-4967-b80d-44f4105e4a2d.png)

The tooltip is used if the email field is empty on submission:

![image](https://user-images.githubusercontent.com/22231637/217094318-e3dff128-7dac-437b-a972-cbf58c23a438.png)